src/lib/registration/ui/addon_selection_reregistration_dialog.rb

```diff
@@ -62,6 +64,8 @@ def addon_selected?(addon)
       # empty implementation, allow reregistration of a dependant addon
       # without reregistering its parent
       def reactivate_dependencies
+        # FIXME: how to test this?
```

Comment:
But the comment points out a naming problem: the method was  actually a blend of a generic hook which called a specific action (reactivating the dependencies), but now it needs 2 actions, adding a refresh of the richtext widget. I'll fix that.